libs/manage/file.ts

```diff
@@ -202,13 +202,20 @@ const updateDetails = function (
     respond(true, errorMessages.UPDATE_DATA_MISSING, callback);
     return;
   }
-  var data = {
-    tags: updateData.tags,
-    customCoordinates: updateData.customCoordinates,
-    extensions: updateData.extensions,
-    webhookUrl: updateData.webhookUrl,
-    customMetadata: updateData.customMetadata
-  };
+
+  var data;
+  if (updateData.publish)
+    data = {
+      publish: updateData.publish,
+    };
+  else
+    data = {
+      tags: updateData.tags,
+      customCoordinates: updateData.customCoordinates,
+      extensions: updateData.extensions,
+      webhookUrl: updateData.webhookUrl,
+      customMetadata: updateData.customMetadata,
+    };
```

Comment:
Similar to https://github.com/imagekit-developer/imagekit-python/pull/69/files#r1745669578

Reply:
Done fixed it